src/Components/TripETA/TripETA.js

```diff
@@ -31,12 +31,12 @@ function TripETA( { modalIsOpen, closeModal } ) {
       <div className='eta-modal'>
         <p className='eta-message'>
           <span>Your ETA for this trip is</span>
-          <span>X hour and Y minutes.</span>
+          {eta && <span>{eta.createTrip.trip.eta} minutes</span>}
```

Comment:
nice work getting the time to generate. Looking forward to styling this like the mod 1 days